R/data.R

```diff
@@ -0,0 +1,18 @@
+#' Ebola linelist data from Fang et al. (2016)
+#'
+#' Linelist data for the Ebola virus collected in Sierra Leone.
```

Comment:
Here might be a good opportunity to ask  people explicitly to cite

Reply:
Yes can do. Is there a nice way to include bibtex in `roxygen2` or I can just write "please cite"

Reply:
you should be able to use a @reference macro I think but not sure what it gets you vs @source. I am really just suggesting saying please cite